---
layout: page
title: Publications
permalink: /publications/
---

I attempt to keep this list up-to-date (last updated: 20 April 2022). More recent publications may be on [Google Scholar](https://scholar.google.com/citations?user=oaQPy0EAAAAJ). Articles should be open access, but links to preprints are included too where available. 


## Preprints 

The mobilome associated with Gram-negative bloodstream infections: A large-scale observational hybrid sequencing based study  
S. Lipworth<sup>+</sup>, W. Matlock<sup>+</sup>, L. P. Shaw, et al. *medrxiv* (2022)  
preprint: [10.1101/2022.04.03.22273290](https://doi.org/10.1101/2022.04.03.22273290)  

Pre-existing chromosomal polymorphisms in pathogenic *E. coli* potentiate the evolution of antibiotic resistance by *MCR-1* plasmid acquisition  
P. K. Jangir, Q. Yang, L. P. Shaw et al. *biorxiv* (2022)  
preprint: [10.1101/2022.02.20.481192](https://doi.org/10.1101/2022.02.20.481192)  


A phylogeny-based metric for estimating changes in transmissibility from recurrent mutations in SARS-CoV-2  
D. Richard, **L. P Shaw**, R. Lanfear, M. Acman, C. Owen, C. C. S. Tan, L. van Dorp, F. Balloux  *biorxiv* (2021)  
preprint: [10.1101/2021.05.06.442903](https://doi.org/10.1101/2021.05.06.442903)


Hospital outbreak of carbapenem-resistant Enterobacterales associated with an OXA-48 plasmid carried mostly by *Escherichia coli* ST399  
A. Ledda, M. Cummins, **L. P. Shaw**, E. Jauneikaite, K. Cole, F. Lasalle, D. Barry, C. Rosmarin, S. Anaraki, D. Wareham,  N. Stoesser, J. Paul, R. Manuel, B. Cherian,  X. Didelot *Microbial Genomics* (in press)  
preprint: [10.1101/2020.06.15.148189](https://doi.org/10.1101/2020.06.15.148189)


## 2022

Role of mobile genetic elements in the global dissemination of the carbapenem resistance gene *bla*NDM  
M. Acman, R. Wang, L. van Dorp,  **L. P. Shaw**, Q. Wang, N. Luhmann, Y. Yin, S. Sun, H. Chen, H. Wang, F. Balloux  *biorxiv* (2020)  
doi: [10.1038/s41467-022-28819-2](https://doi.org/10.1038/s41467-022-28819-2)  
preprint: [10.1101/2021.01.14.426698](https://doi.org/10.1101/2021.01.14.426698)  


## 2021

A genomic epidemiological study shows prevalence of antimicrobial resistance in Enterobacterales is associated with the livestock host, as well as antimicrobial usage  
M. AbuOun, H. Jones, E. Stubberfield, D. Gilson, **L. P. Shaw**, A. Hubbard, K. Chau, R. Sebra, T. Peto, D. Crook, D. Read, H. S. Gweon, A. S. Walker, N. Stoesser, R. P. Smith, M. Anjum *Microbial Genomics* (in press)  
doi: [10.1099/mgen.0.000630](https://doi.org/10.1099/mgen.0.000630)  
accepted version: [here](https://ora.ox.ac.uk/objects/uuid:a8585f70-22e2-4370-b962-277e0b432aa1)


Flanker: a tool for comparative genomics of gene flanking regions  
W. Matlock<sup>+</sup>, S. Lipworth⁺, B. Constantinides, T. E. A. Peto, A. S. Walker, D. Crook, S. Hopkins, **L. P. Shaw**⁺, N. Stoesser⁺ *Microbial Genomics* (2021)  
doi: [10.1099/mgen.0.000634](https://doi.org/10.1099/mgen.0.000634)  
preprint: [10.1101/2021.02.22.432255](https://doi.org/10.1101/2021.02.22.432255)


Drivers of methicillin resistant *Staphylococcus aureus* (MRSA) lineage replacement in China  
H. Chen⁺, Y. Yin⁺, L. van Dorp, **L. P. Shaw**, H. Gao, M. Acman, J. Yuan F. Chen, S. Sun, X. Wang, S. Li, Y. Zhang, R. A. Farrer, H. Wang, F. Balloux *Genome Medicine* (2021)
doi: [10.1186/s13073-021-00992-x](https://doi.org/10.1186/s13073-021-00992-x)  


Letter: Null models for gene enrichment in plasmids  
**L. P. Shaw** *PNAS* (2021)  
doi: [10.1073/pnas.2103665118](https://doi.org/10.1073/pnas.2103665118)


Niche and local geography shape the pangenome of wastewater- and livestock-associated Enterobacteriaceae  
**L. P. Shaw** et al. on behalf of the [REHAB consortium](http://modmedmicro.nsms.ox.ac.uk/rehab/) *Science Advances*  (2021)  
doi: [10.1126/sciadv.abe3868](https://doi.org/10.1126/sciadv.abe3868)  
preprint: [10.1101/2020.07.23.215756](https://www.biorxiv.org/content/10.1101/2020.07.23.215756)  


Genomic network analysis of an environmental and livestock IncF plasmid population  
W. Matlock, ..., A. S. Walker⁺, N. Stoesser⁺, **L. P. Shaw**⁺ on behalf of the [REHAB consortium](http://modmedmicro.nsms.ox.ac.uk/rehab/) *The ISME Journal*   
doi: [10.1038/s41396-021-00926-w](https://doi.org/10.1038/s41396-021-00926-w)  
preprint: [10.1101/2020.07.24.215889](https://doi.org/10.1101/2020.07.24.215889)


Electronic application to improve management of infections in low-income neonatal units: pilot implementation of the NeoTree beta app in a public sector hospital in Zimbabwe  
H. Gannon, ... [12 authors inc. L. P. Shaw], F. C. Fitzgerald *BMJ Open Quality* (2021)  
doi: [10.1136/bmjoq-2020-001043](https://doi.org/10.1136/bmjoq-2020-001043)


## 2020

No evidence for increased transmissibility from recurrent mutations in SARS-CoV-2  
L. van Dorp⁺, D. Richard⁺,  C. CS. Tan, **L. P. Shaw**, M. Acman, F. Balloux *Nature Communications*   
doi: [10.1038/s41467-020-19818-2](https://doi.org/10.1038/s41467-020-19818-2)   
preprint: [10.1101/2020.05.21.108506v5](https://www.biorxiv.org/content/10.1101/2020.05.21.108506)

STROBE-Metagenomics: A STROBE/STROME-ID extension statement to guide the reporting of studies applying metagenomics  
T. Bharucha, C. Oeser, [alphabetical: 23 authors inc. L. P. Shaw], J. Breuer, N. Field *Lancet Infectious Diseases* 20:e251–60  
doi: [10.1016/S1473-3099(20)30199-7](https://doi.org/10.1016/S1473-3099(20)30199-7)

The phylogenetic range of bacterial and viral pathogens of vertebrates  
**L. P. Shaw**⁺, A. Wang⁺, et al. *Molecular Ecology* 29:3361–3379   
doi: [10.1111/mec.15463](https://doi.org/10.1111/mec.15463)   
preprint: [10.1101/670315v3](https://doi.org/10.1101/670315)  

Emergence of genomic diversity and recurrent mutations in SARS-CoV-2  
L. van Dorp, M. Acman⁺, D. Richard⁺, **L. P. Shaw**⁺, C. E. Ford, L. Ormond, C. J. Owen, J. Pang, C. CS. Tan, F. AT. Boshier, A. Torres Ortiz, F. Balloux *Infection, Genetics and Evolution* 104351  
doi: [10.1016/j.meegid.2020.104351](https://doi.org/10.1016/j.meegid.2020.104351)

Discordant bioinformatic predictions of antimicrobial resistance from whole-genome sequencing data of bacterial isolates: an inter-laboratory study  
R. M. Doyle, D. M. O'Sullivan, [alphabetical: 16 authors inc. L. P. Shaw] et al. *Microbial Genomics* 6(2)   
doi: [10.1099/mgen.0.000335](https://doi.org/10.1099/mgen.0.000335)  
preprint: [10.1101/793885v2](https://doi.org/10.1101/793885)  

Genomic diversity affects the accuracy of bacterial SNP calling pipelines  
S. J. Bush, D. Foster, D. W. Eyre, E. L. Clark, N. De Maio, **L. P. Shaw** et al. *GigaScience* 9(2):giaa007   
doi: [10.1093/gigascience/giaa007](https://doi.org/10.1093/gigascience/giaa007)  
preprint: [10.1101/653774v1](https://doi.org/10.1101/653774)  

## 2019

The impact of sequencing depth on the inferred taxonomic composition and AMR gene content of metagenomic samples  
H. G. Soon, **L. P. Shaw** et al. on behalf of the [REHAB consortium](http://modmedmicro.nsms.ox.ac.uk/rehab/). *Environmental Microbiome* 14:7  
doi: [10.1186/s40793-019-0347-1](https://doi.org/10.1186/s40793-019-0347-1)  
preprint: [10.1101/593301v1](https://doi.org/10.1101/593301)  

Comparison of long-read sequencing technologies in the hybrid assembly of complex bacterial genomes  
N. De Maio⁺, **L. P. Shaw**⁺ et al. on behalf of the [REHAB consortium](http://modmedmicro.nsms.ox.ac.uk/rehab/). *Microbial Genomics* 5(9)  
doi: [10.1099/mgen.0.000294](https://doi.org/10.1099/mgen.0.000294)  
preprint: [10.1101/530824v2](https://doi.org/10.1101/530824)  

Representing antibiotic relationships using measurements of efficacy against clinical isolates [version 3; peer review: 2 approved]  
**L. P. Shaw**. *Wellcome Open Research* 4:86  
doi: [10.12688/wellcomeopenres.15304.3](https://doi.org/10.12688/wellcomeopenres.15304.3)

Whole genome sequencing *Mycobacterium tuberculosis* directly from sputum identifies more genetic diversity than sequencing from culture  
C. Nimmo, **L. P. Shaw**, et al. *BMC Genomics* 20:389  
doi: [10.1186/s12864-019-5782-2](https://doi.org/10.1186/s12864-019-5782-2)  
preprint: [10.1101/446849v3](https://doi.org/10.1101/446849)  

Modelling microbiome recovery after antibiotics using a stability landscape framework  
**L. P. Shaw**, H. Bassam, C. P. Barnes, N. Klein, A. Sarah Walker, F. Balloux. *The ISME Journal* 13:1845  
doi: [10.1038/s41396-019-0392-1](https://doi.org/10.1038/s41396-019-0392-1)  
preprint: [10.1101/222398v3](https://doi.org/10.1101/222398)  

Children with cystic fibrosis are infected with multiple subpopulations of *Mycobacterium abscessus* with different antimicrobial resistance profiles  
**L. P. Shaw**, R. Doyle, E. Kavaliunaite, H. Spencer, F. Balloux, G. Dixon, K. Harris. *Clinical Infectious Diseases* 69(10):1678  
doi: [10.1093/cid/ciz069](https://doi.org/10.1093/cid/ciz069)  
preprint: [10.1101/436261v1](https://doi.org/10.1101/436261v1)  

Review: The microbiome — the explanation for (almost) everything?  
**L. P. Shaw** and N. Klein. *The Pediatric Infectious Disease Journal* 38(4):e69  
doi: [10.1097/INF.0000000000002261](https://doi.org/10.1097/INF.0000000000002261)


## 2018

Portable sequencing and the scales of genomic data in global EID surveillance  
**L. P. Shaw**⁺ and N. Sugden⁺. *Geo* 5(2):e00066   
doi: [10.1002/geo2.66](https://doi.org/10.1002/geo2.66)

Microbial translocation does not drive immune activation in Ugandan children with HIV  
F. C. Fitzgerald, E. L'homme, K. Harris, J. Kenny, R. Doyle, C. Kityo, **L. P. Shaw** et al. *Journal of Infectious Diseases* 219(1):89  
doi: [10.1093/infdis/jiy495](https://doi.org/10.1093/infdis/jiy495)

Review: From theory to practice: translating whole-genome sequencing (WGS) into the clinic  
F. Balloux⁺, O. B. Brynildsrud⁺, L. van Dorp⁺, **L. P. Shaw** et al. *Trends in Microbiology* 26(12):P1035  
doi: [10.1016/j.tim.2018.08.004](https://doi.org/10.1016/j.tim.2018.08.004)

The global distribution and spread of the mobilized colistin resistance gene *mcr-1*  
R. Wang⁺, L. van Dorp⁺, **L. P. Shaw**⁺ et al. *Nature Communications* 9:1179  
doi: [10.1038/s41467-018-03205-z](https://doi.org/10.1038/s41467-018-03205-z)  
preprint: [10.1101/220921](https://doi.org/10.1101/220921)  

Periapical infection may affect birth outcomes via systemic inflammation  
U. Harjunmaa, R. Doyle, J. Jornstedt, S. Kamiz, J.M. Jorgensen, C.P. Stewart, **L. P. Shaw**, et al. *Oral Diseases* 24(5):847  
doi: [10.1111/odi.12817](https://doi.org/10.1111/odi.12817)

Deep sequencing of oral microbiomes from hunter-gatherers and traditional agriculturalists reveals shifts in commensal taxon balance and pathogen load linked to subsistence strategy  
F. Lassalle⁺, M. Spagnoletti⁺, M. Fumagalli, **L. P. Shaw**, et al. *Molecular Ecology* 27(1):182  
doi: [10.1111/mec.14435](https://doi.org/10.1111/mec.14435)


## 2017

 The human salivary microbiome is shaped by shared environment rather than genetics: evidence from a large family of closely related individuals  
 **L. P. Shaw**⁺, A. Ribeiro⁺, A. Levine, N. Pontikos, F. Balloux, A. Segal, A. Roberts, A. Smith. *mBio* 8:e01237  
 doi: [10.1128/mBio.01237-17](https://doi.org/10.1128/mBio.01237-17)

Review: The oral microbiome  
**L. P. Shaw**, A. Smith, A. Roberts. *Emerging Topics in Life Sciences* 1(4):287  
doi: [10.1042/ETLS20170040](https://doi.org/10.1042/ETLS20170040)

Transient structural variations have strong effects on quantitative traits and reproductive isolation in fission yeast  
D. Jeffares, C. Jolly, M. Hoti, D. Speed, C. Rallis, **L. P. Shaw**, F. Balloux, C. Dessimoz, J. Bahler, F. Sedlazeck. *Nature Communications* 8:14061  
doi: [10.1038/ncomms14061](https://doi.org/10.1038/ncomms14061)  
preprint: [10.1101/047266](https://doi.org/10.1101/047266)  

## 2016

Distinguishing the signals of gingivitis and periodontitis in supragingival plaque: a cross-sectional cohort study in Malawi  
**L. P. Shaw**, U. Harjunmaa, R. Doyle et al. *Applied and Environmental Microbiology* 82:6057  
doi: [10.1128/AEM.01756-16](https://doi.org/10.1128/AEM.01756-16)

Opinion: Claims made in scientific press releases need greater accountability  
**L. P. Shaw**. *BMJ* 2016;353:i2728  
doi: [10.1136/sbmj.i2728](https://doi.org/10.1136/sbmj.i2728)

## 2015

DNA hairpins destabilize duplexes primarily by promoting melting rather than by inhibiting hybridization  
J. Schreck, T. Ouldridge, F. Romano, P. Sulc, **L. P. Shaw**, A. Louis, J. Doye. *Nucleic Acids Research* 43(13):6181  
doi: [10.1093/nar/gkv582](https://doi.org/10.1093/nar/gkv582)
