---
layout: page
title: Research
permalink: /research/
---

### Bacterial genomics

The increasing availability of [complete bacterial genomes](https://doi.org/10.1099/mgen.0.000294) allows the investigation of the processes which underpin bacterial evolution. I'm particularly interested in the spread of mobile genetic elements which move ['horizontally'](https://doi.org/10.1038/s41467-018-03205-z) across bacterial genomes and even species, and untangling this nested genetic complexity. These elements include plasmids as well as smaller transposons which can carry genes for antimicrobial resistance.

### Microbial ecology

The human microbiome is the collective name given to the microbial communities that live in our bodies. Just as you have ecology for animals, you can have it for microbes too. I've previously worked on the microbiome's composition in [health](https://doi.org/10.1128/mBio.01237-17) and [disease](https://doi.org/10.1128/AEM.01756-16). One topic that I'm still interested in is understanding the effect of antibiotics: how they disrupt microbial communities and how the community [recovers](https://doi.org/10.1101/222398).

### Antimicrobial resistance

My current research focuses on understanding the processes behind the development and subsequent spread of antimicrobial resistance using genomic data. This can be at multiple scales, from [within patients](https://doi.org/10.1093/cid/ciz069) to [global spread](https://doi.org/10.1038/s41467-018-03205-z). I've previously been involved in efforts to improve the availability of surveillance data of antimicrobial resistance, including carrying out work awarded a [Data Reuse Prize](https://doi.org/10.1186/s12864-019-5782-2) by Wellcome.  

## Other stuff

Some papers in this miscellaneous category have included [host-pathogen biology](https://doi.org/10.1101/670315), [science and technology studies](https://doi.org/10.1002/geo2.66), and [yeast genetics](https://doi.org/10.1038/ncomms14061).
